src/components/submission-form/Submission-form.tsx

```diff
@@ -46,6 +46,7 @@ export function SubmissionForm() {
   const { data: session, status } = useSession();
   const [isBusy, setIsBusy] = useState(false);
   const [isSubmitted, setIsSubmitted] = useState(false);
+  const [cancelModalOpen, setCancelModalOpen] = useState(false);
```

Comment:
little complicated naming convention. I prefer to use openModal - setOpenModal - it's simple and you don't need to combine cancel and open in one name

Reply:
Updated